index.bs

```diff
@@ -271,7 +374,40 @@ interface FileSystemFileHandle : FileSystemHandle {
 };
 </xmp>
 
-A {{FileSystemFileHandle}}'s associated [=FileSystemHandle/entry=] must be a [=file entry=].
+A {{FileSystemFileHandle}}'s associated [=FileSystemHandle/locator=]'s
+[=file system locator/kind=] is {{FileSystemHandleKind/"file"}}.
+
+<div algorithm>
+To <dfn data-lt="creating a child FileSystemFileHandle">create a child FileSystemFileHandle</dfn>
```

Comment:
Since FileSystemFileHandle is code, we should still format it as such with `<code>`. Also from the places that call this.

Reply:
Hmm looks like we do the same thing for FileSystemWritableFileStream and FileSystemSyncAccessHandle. I'll fix those in a follow-up